### DISCUSSION TITLE
Paid icons in Miro templates

### DISCUSSION THREAD

**Discussion by yellowbrickc** [ddd-crew/ddd-starter-modelling-process#53] [Team Posts]
Hi @ddd-crew/crew,

@Max-Git replaced the paid icons in one repo ([here the PR](https://github.com/ddd-crew/virtual-modelling-templates/pull/1)).
Cool thing, I would not know, which are paid and which are free 🤔 I think we should take care to not add new paid icons but for this, we should know, which are which

Here my questions regarding this:
1. @Max-Git how did you know?
2. are we sure that all the icons we are using are free?

Cheers,
Krisztina

**Comment by Max-Git** [comment]
@yellowbrickc you can tell by the icon background stripes when you zoom in.
Miro does not allow to filter icon search results with free only icons. You need to go to iconfinder.com and filter results to be sure to use free icons.

**Comment by yellowbrickc** [comment]
Boah. This sound like laborious work. Maybe could each of you check his own templates and change it if is necessary?

**Comment by Max-Git** [comment]
Sounds like a good option while waiting for Miro to add a free icons only checkbox!
